packages/stable/src/api/geospatial/types.ts

```diff
@@ -65,20 +65,20 @@ export interface FeatureTypes {
   searchSpec?: {
     [searchName: string]: {
       attributes:
-        | Array<keyof FeatureTypesCreateItem['attributes']>
+        | Array<keyof FeatureTypeCreateItem['attributes']>
         | ['_created_at']
         | ['_updated_at']
         | ['_external_id'];
     };
   };
 }
 
-export interface FeatureTypesCreateItem {
+export interface FeatureTypeCreateItem {
   externalId: CogniteExternalId;
```

Comment:
ExternalId?

Reply:
extended ExternalId interface. 
I'm hoping that is what you were looking for here.